webpack.config.js

```diff
@@ -31,9 +31,37 @@ module.exports = (env, argv) => {
     var outfile = 'skulpt.js';
     var assertfile = './assert-dev.js';
     var mod = {};
+    var externals = {};
+    var plugins = [
+        new CleanWebpackPlugin(),
+        new CopyWebpackPlugin([
+            { from: 'debugger/debugger.js', to: 'debugger.js' }
+        ]),
+        new webpack.DefinePlugin({
+            GITVERSION: JSON.stringify(git.version()),
+            GITHASH: JSON.stringify(git.commithash()),
+            GITBRANCH: JSON.stringify(git.branch()),
+            BUILDDATE: JSON.stringify(new Date())
+        }),
+        new CompressionWebpackPlugin({
+            include: /^skulpt\.min\.js$/,
+            algorithm: 'gzip'
+        }),
+    ];
+
 
     if (argv.mode === 'production') {
         opt = {
+            splitChunks: {
+                cacheGroups: {
```

Comment:
Does this mean the bitinterger.js is split off into vendor.js?

Reply:
Yes.  biginteger, strftime, strptime, and setimmediate all end up in vendor.js.  None of them can get through the closure compiler.